src/components/Navbar.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+function Navbar() {
+  return (
+    <div>
+      <nav className="navbar navbar-dark bg-primary mb-3">
+        <div className="container">
+          <a className="navbar-brand" href="/">WikiCountries</a>
```

Comment:
Remember that we don't use `<a>` anymore. You should use `<Link>` and also import it from `'react-router-dom'`